integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/common/ActionUtils.java

```diff
@@ -0,0 +1,20 @@
+package com.redhat.service.bridge.integration.tests.common;
+
+import io.vertx.core.json.JsonObject;
+
+public class ActionUtils {
+
+    public final static String SLACK_URI = "https://slack.com/api/conversations.history";
+    public final static String SLACK_TOKEN = "xoxb-3121370458964-3137484984228-u7b4MFP7xu8QRGmYswhx8w94";
```

Comment:
should Slack token be private?

Reply:
Sure will modify

Reply:
if you can delete this token and generate a new one it would be great